src/components/Inputs/TaxIdInput/TaxIdInput.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+  <div class="pb-document-input-container">
+    <input
+      v-model="taxIdInput"
+      class="pb"
+      style="width: 100%"
+      :class="validationClass"
+      :style="taxIdInputStyle"
+      :placeholder="setInputPlaceholder"
+      :maxlength="getMaxLength(inputType)"
+      @blur="validateDocument"
+      @focus="updateValidationFiled"
+    >
+    <p
+      v-if="state.message"
+      class="pb error-text"
+    >
+      {{ state.message }}
+    </p>
+  </div>
+</template>
+
+<script>
+import { validateColor } from '@pb/utils/validator';
+import { documentPlaceholders } from '@pb/utils/inputDocumentTypes';
+import { isCpf, isCnpj } from 'adapcon-utils-js';
+
+export default {
+  name: 'PbTaxIdInput',
+
+  props: {
+    required: { type: Boolean, default: false },
+    value: { type: String, default: '' },
+    background: { type: String, default: 'transparent' },
+    color: { type: String, default: 'gray-20', validator: validateColor },
+    inputType: {
+      type: Array,
+      default: () => ['cnpj'],
+    },
+    placeholder: { type: String, default: '__.___.___/____-__' },
+  },
+
+  data() {
+    return {
+      state: {
+        validation: true,
+        message: '',
+        inputValue: '',
+      },
+    };
+  },
+
+  computed: {
+    validationClass() {
+      return !this.state.validation ? 'error' : '';
+    },
+    taxIdInputStyle() {
+      return {
+        ...(this.state.validation
+          ? {
+            background: `var(--color-${this.background})`,
+            border: `1px solid var(--color-${this.color})`,
+          }
+          : { background: `var(--color-${this.background})` }),
+      };
+    },
+
+    setInputPlaceholder() {
+      if (!this.placeholder && this.inputType.length <= 1)
```

Comment:
The placeholder prop have a default value... So this if statement can validate just the this.inputType.length <= 1 condition.

```suggestion
      if (this.inputType.length <= 1)
```

Reply:
I don´t think so. In case when we´re using two or more types of documents, this should work if we remove the placeholder validation, when using only one type of input, we need to validate if there´s no placeholder value setted to return the default value. ('__.___.___/____-__' or '___.___.___-__'). So in this case, when we remove the placeholder validation it always return this default values, and the user will be not able to set a custom one.

Reply:
Got it! 🚀